sensors/sensors_test.go

```diff
@@ -205,7 +207,11 @@ func TestTimedLidarSensorReading(t *testing.T) {
 		tsr, err := goodReplayLidar.TimedLidarSensorReading(ctx)
 		test.That(t, err, test.ShouldBeNil)
 		test.That(t, tsr.Reading, test.ShouldNotBeNil)
-		test.That(t, tsr.ReadingTime, test.ShouldEqual, time.Date(2006, 1, 2, 15, 4, 5, 999900000, time.UTC))
+
+		readingTime, err := time.Parse(time.RFC3339Nano, s.TestTimestamp)
```

Comment:
[note] remove magic number and ensure equality by parsing/formatting the same way